packages/environment-check/src/checks/get-installed.ts

```diff
@@ -4,10 +4,12 @@ import { join } from 'path';
 import { spawnCommand, npmCommand } from '../command';
 import { Extensions, NpmModules } from '../types';
 import type { ILogger } from '../types';
+import type { Extension } from 'vscode';
 import { isAppStudio } from '@sap-ux/btp-utils';
 
 const pluginsDirBAS = join('/extbin/plugins');
 const globalNodeModulesBAS = join('/extbin/npm/globals/lib/node_modules');
+const globalNodeModulePnpmBAS = join('/extbin/globals/pnpm/5/node_modules');
```

Comment:
Can we please determine this path programmatically. This is specific to a pnpm version and the global loc can be changed by config.